src/compas/datastructures/mesh/operations/join.py

```diff
@@ -4,7 +4,7 @@
 
 from compas.utilities import geometric_key
 
-from compas.geometry import mesh_cull_duplicate_vertices
+from compas.geometry.algorithms.purging import mesh_cull_duplicate_vertices
```

Comment:
not importable from `compas.geometry`?

Reply:
it works when I use the function but the checks fail

Reply:
fixed a few imports. try again, i think it will work now...